File: asg32/Question3.java
import java.util.*;

class StringDemo {
    public String str = null;

    public StringDemo(String str) {
        this.str = str;
    }

    public int CountDiff() {
        char Arr[] = str.toCharArray();
        int iCnt = 0;
        int iCountCapital = 0;
        int iCountSmall = 0;
        for (iCnt = 0; iCnt < Arr.length; iCnt++) {
            if (Arr[iCnt] >= 'a' && Arr[iCnt] <= 'z') {
                iCountCapital++;
            } else if (Arr[iCnt] >= 'A' && Arr[iCnt] <= 'Z') {
                iCountSmall++;
            }

        }
        return iCountCapital - iCountSmall;

    }
}

class Question3 {
    public static void main(String args[]) {
        Scanner sobj = new Scanner(System.in);
        String str = null;
        int iRet = 0;

        System.out.println("Enter the String : ");
        str = sobj.nextLine();

        StringDemo SDobj = new StringDemo(str);

        iRet = SDobj.CountDiff();
        System.out.println("Difference between capital and small charachter are : " + iRet);

        sobj.close();

    }
}

File: asg50/Question3.c
#include<stdio.h>

int Small(char *str){
    static int count=0;
    if (*str!='\0')
    {
        if(*str>='a' && *str<='z'){
            count++;
        }
        str++;
        Small(str);
    }
    return count;
}

int main(){
    int iRet=0;
    char arr[20];
    printf("Enter string : \n");
    scanf("%[^'\n']s",&arr);

    iRet=Small(arr);
    printf("Small : %d\n",iRet);


    return 0;
}